yb-voyager/cmd/migration_complexity.go

```diff
@@ -200,3 +210,127 @@ func getComplexityForLevel(level string, count int) string {
 		panic(fmt.Sprintf("unknown impact level %s for determining complexity", level))
 	}
 }
+
+// ======================================= Migration Complexity Explanation ==========================================
+
+// TODO: discuss if the html should be in main report or here
+const explainTemplateHTML = `
+<p>Below is a detailed breakdown of issues by category, showing the count for each impact level.</p>
```

Comment:
How about `Below is a breakdown of the issues detected in different categories for each impact level` ?

Reply:
yeah better.